jenkins/requirements.txt

```diff
@@ -1 +0,0 @@
-requests==2.5.3
```

Comment:
Ahh, I see what you are saying. If you could check that would be great (I would check but I wouldn't even know where to look?). It does seem like we should break out a top level ``requirements.txt`` from ``test-requirements.txt`` and have scripts just install from that until we all get on a pip that can deal with the same requirement included multiple times.